BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.h

```diff
@@ -77,6 +77,12 @@ typedef NS_ENUM(NSUInteger, BOXContentSDKStreamError) {
     BOXContentSDKStreamErrorReadFailed = 30001
 };
 
+typedef NS_ENUM(NSUInteger, BOXContentSDKCacheError) {
+    BOXContentSDKSCacheErrorArchiveFailed = 40000,
+    BOXContentSDKSCacheErrorFileCreateFailed = 40001,
+    BOXContentSDKSCacheErrorNoValidSessionTaskGivenUserIdAssociateId = 40002
```

Comment:
Minor: The naming for the last one is a bit confusing. Is `BOXContentSDKCacheErrorNoValidSessionForUserId` inaccurate?